metron-platform/metron-common/src/test/java/org/apache/metron/common/stellar/StellarTest.java

```diff
@@ -285,6 +285,18 @@ public void testHappyPath() {
   }
 
   @Test
```

Comment:
Yes, empty map.  If a variable does not exist, Stellar will pass in a null as the variable.